recipe/meta.yaml

```diff
@@ -10,8 +10,9 @@ source:
 
 build:
   number: 1001
-  # mpfr only supports vc14
-  skip: true  # [win and py<35]
+  # - mpfr only supports vc14
+  # - SWIG bindings have a python linkage problem in macOS
+  skip: true  # [(win and py<35) or osx]
```

Comment:
that's the easy way out, the right fix is to pass --undefined-lookup to the linker

Reply:
See my comment [below](https://github.com/conda-forge/cgal-feedstock/pull/41#issuecomment-474976230). I think we should release this for Windows & Linux already, the update is stalled for many months now. I don't have a quick way to locally test macOS builds, but please, feel free to make a PR and add yourself as a maintainer if you wish ;)

Reply:
the definitive fix should probably be done in `cgal-swig-bindings`